tidb/run.sh

```diff
@@ -2,18 +2,29 @@
 
 rm failed.log
 
-for test in "bank" "sets" "register"
+for test in "bank" "sets"
 do
     for nemesis in "none" "parts" "majority-ring" "start-stop-2" "start-kill-2"
         do
-	        lein run test --test ${test} --nemesis ${nemesis} --time-limit 300 --recovery-time 30 --concurrency 10
+	        echo lein run test --test ${test} --nemesis ${nemesis} --time-limit $t --concurrency 10
             if [ $? -ne 0 ]
             then
                 echo ${test} ${nemesis} >> failed.log
             fi
     done
 done
 
+for nemesis in "none" "parts" "majority-ring" "start-stop-2"
+do
+    if [ ${nemesis} = "start-stop-2" ]
```

Comment:
this loop is for register test, it might run out of memory if we choose 180, i will try it anyway.